NAMESPACE

```diff
@@ -21,10 +22,14 @@ exportClasses(LogLogisticOs)
 exportClasses(LongModel)
 exportClasses(OsModel)
 exportClasses(StanModule)
+exportClasses(jmModel)
```

Comment:
better name would be `JointModel` because `jm` is already abbreviating "Joint Model" and the others end with "Model"

Reply:
Nice observation. I will fix it.